internal/node-patches/tslint.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "gts/tslint.json",
+  "linterOptions": {
```

Comment:
I would expect linting options to be uniform over the whole monorepo

Reply:
Ideologically this is a package. so it's gets all of it's own stuff. 
Also this is generated from google typescript style https://www.npmjs.com/package/gts which we don't use anywhere else in this repo but we should because we're google and use typescript.

We can drop in the commands to use the repo's linter/formatter if we're not interested in changing typescript code styles.

Reply:
oh, gts is from us, I didn't realize that. We should talk more about whether to adopt it generally, not just in one package

"google style" in practice is to have one set of configs for a whole monorepo, not one per package. we'll have to reconcile the ideologies

Reply:
this is google style mapped to a world with versions. turns out it has to become a dependency but that lets it apply to everything. this is clearly not the place to convince anyone about the goodness or not of either approach. 

This particular package (gts) doesn't work for non node shaped typescript projects so there is no way to reach a useful consensus except to agree that there is a place in rules_nodejs where we can write nodejs style packages that can be used in internal, or not.

The existing linter works for any shape of typescript and js. the only downside is its not a dependency of the package so the node package loses its Ideological boundary.  Because it's an ideological boundary and this is a monorepo and javascript we can freely let it go. 

The npm script to lint should just run the husky command and expect it works because it's in the monorepo.